File: Adapter/globits-hiv-info-adapter/GlobitsHivInfoAdapter/Dto/SerializeObjectDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GlobitsHivInfoAdapter.Dto
{
    public class SerializeObjectDto
    {
        public DateTime dateGetData { get; set; }
        public DateTime timeStartJob { get; set; }
        public int totalRecord { get; set; }
        public bool status = false;
    }
}

File: Adapter/opc-eclinica-adapter/opc-eclinica-adapter-api/src/main/java/com/globits/adapter/eclinica/data/dto/SerializeObjectDto.java
package com.globits.adapter.eclinica.data.dto;

import java.io.Serializable;
import java.util.Date;

public class SerializeObjectDto implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private Date dateGetData;
	private Date timeStartJob;
	private long totalRecord;
	private Boolean status;
	public Date getDateGetData() {
		return dateGetData;
	}
	public void setDateGetData(Date dateGetData) {
		this.dateGetData = dateGetData;
	}
	public Date getTimeStartJob() {
		return timeStartJob;
	}
	public void setTimeStartJob(Date timeStartJob) {
		this.timeStartJob = timeStartJob;
	}
	
	public Boolean getStatus() {
		return status;
	}
	public void setStatus(Boolean status) {
		this.status = status;
	}
	public long getTotalRecord() {
		return totalRecord;
	}
	public void setTotalRecord(long totalRecord) {
		this.totalRecord = totalRecord;
	}
	
}
